docs/guide/production.md

```diff
@@ -0,0 +1,61 @@
+# Running for production
+In this section, we will talk about how to run a node for production. At the moment we will focus on Linux (Redhat Compatiable) only.
+
+`terrad` does not require super user account. We **strongly** recommend to run `terrad` as a normal user. However, at the configuration, we need super user permission for creating/modifying files.
+
+This guide is for general purpose only. We recommend to read [Validator](https://docs.terra.money/features/overview) section for operating validator node.
+
+## Firewall configuration
+`terrad` uses several TCP port for each purposes.
+
+* `26656` is default port for P2P protocol. This port has to be opened to communicate with other nodes. This port must be opened for joining the network. **However,** it does not have to be opened to public. For validator nodes, we recommened to configure `persistent_peers` and close this port to public.
```

Comment:
```suggestion
* `26656` is the default port for P2P protocol. This port has to be opened to communicate with other nodes. This port must be opened for joining the network. **However,** it does not have to be opened to the public. For validator nodes, we recommend to configure `persistent_peers` and close this port to the public.
```
This part is a bit confusing for me.
Is this port used only for peers discovery?
In this sentence it sounds like it's used for communicating data with other nodes.